exercises/04_power/power.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func Pow(x, y uint) (uint, uint) {
+	if y == 0 {
```

Comment:
Please remove the `if` block. Only for loop is enough (you will need to modify your for-loop version though).